src/views/Home.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+<div id="app1">
+<div class="body">
+  <div class="step-box">
+    <span>STEP1</span>
+    <h4>お客様情報を入力してください</h4>
+  </div>
+  <div class="question-box">
+    <p>-性別-</p>
+    <div class="boxes">
+      <label><input type="radio" v-model="radio" value="yes" >男性</label>
+      <label><input type="radio" v-model="radio" value="no" >女性</label>
+    </div>
+  </div>
+  <br>
+  <div class="question-box">
+    <p>-生年月日-</p>
+    <div class="boxes">
+    <select id="year" v-model="year">
+    <option v-for="nengo in nengoes" :key="nengo.year" :value="nengo.year">{{ nengo.label }}</option>
+    </select>
+    <label>年</label>
+    <select v-model="month" @change="get_days">
+    <option v-for="n in 12" :value="n" :key="n">{{ n }}</option>
+    </select>月
+    <select v-model="day">
+    <option v-for="n in 31" :value="n" :key="n">{{ n }}</option>
+    </select>日
+    </div>
+  </div>             
+</div>
+<router-link to="/about"><button class="button is-primary" id="next-btn">次へ進む</button></router-link>
+</div>
+
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      year: 2000,
+      month: 1,
+      date: 1,
+      day: 1,
+      radio:'',
+      radio2:'',
+      radio3:'',
+      nengoes: []
+    };
+  },
+  mounted() {
+    this.nengoes = this.genereate();
+  },
+  methods: {
+    genereate() {
+      const nengoes = [];
+      for (let y=2020; y>1920; y--) {
+        if (y > 2018) {
+          nengoes.push( {"year": y, "label": `${y} (令和${y-2018}年)`} );
```

Comment:
json形式でpushしていますが、オブジェクトとして定義するのが一般的です。
```suggestion
          nengoes.push( {year: y, label: `${y} (令和${y-2018}年)`} );
```